client/rest-high-level/src/test/java/org/elasticsearch/client/IndicesClientIT.java

```diff
@@ -54,15 +56,34 @@ public void testDeleteIndex() throws IOException {
         }
     }
 
+    public void testOpenExistingIndex() throws IOException {
+        String indexName = "test_index";
+        createIndex(indexName);
+
+        OpenIndexRequest openIndexRequest = new OpenIndexRequest(indexName);
+        OpenIndexResponse openIndexResponse = execute(openIndexRequest, highLevelClient().indices()::openIndex,
+                highLevelClient().indices()::openIndexAsync);
+        assertTrue(openIndexResponse.isAcknowledged());
```

Comment:
maybe it would be good to close the index first using the low-level REST client, then verify that opening it actually does something?